.eslintrc.json

```diff
@@ -12,9 +12,9 @@
         "prettier/@typescript-eslint"
     ],
     "rules": {
-        "no-debugger": "error",
+        "no-debugger": "warn",
         "no-multiple-empty-lines": "error",
-        "react-hooks/exhaustive-deps": "off",
```

Comment:
I don't love not having exhaustive-deps but this should be off or all our PRs will have a ton of warnings